engine/src/main/java/org/destinationsol/game/BeaconHandler.java

```diff
@@ -183,7 +183,7 @@ private void updateD(SolGame game) {
                 farDrawable = (FarDrawable) fo;
                 return;
             }
-            throw new AssertionError();
+            throw new AssertionError("The far object data in " + game + " is a FarDrawable object, does not contain 3 elements in drawables, or the first drawable isnt attackSprite");
```

Comment:
I'm pretty sure `game` is not a string? Even so, printing it out is not useful as I believe there is always only one instance of `SolGame`